ci/kokoro/windows/build.ps1

```diff
@@ -64,12 +67,28 @@ if (($BuildName -eq "cmake") -or ($BuildName -eq "cmake-debug")) {
 } elseif ($BuildName -eq "quickstart-bazel") {
     $DependencyScript = "build-bazel-dependencies.ps1"
     $BuildScript = "build-quickstart-bazel.ps1"
+} elseif ($BuildName -eq "quickstart-cmake-static") {
+    $env:CONFIG = "Debug"
+    $env:GENERATOR = "Ninja"
+    $env:VCPKG_TRIPLET = "x64-windows-static"
+    $env:BUILD_CACHE = "gs://cloud-cpp-kokoro-results/build-cache/google-cloud-cpp/master/vcpkg/Debug/x64-windows-static/quickstart.zip"
+    $DependencyScript = "build-cmake-dependencies.ps1"
+    $DependencyScriptArgs=@("cmake-out/vcpkg-quickstart", "google-cloud-cpp")
+    $BuildScript = "build-quickstart-cmake.ps1"
+} elseif ($BuildName -eq "quickstart-cmake-dll") {
+    $env:CONFIG = "Debug"
+    $env:GENERATOR = "Ninja"
+    $env:VCPKG_TRIPLET = "x64-windows"
+    $env:BUILD_CACHE = "gs://cloud-cpp-kokoro-results/build-cache/google-cloud-cpp/master/vcpkg/Debug/x64-windows/quickstart.zip"
```

Comment:
The static cache contains "x64-windows-static". Do you want this one to contain "-dll" in the name? Or is the _default_ assumed to be dll?

Reply:
I named it after the triplet (`x64-windows`) and yes, the default is DLL.